### ISSUE TITLE
EDDI not functioning

### ISSUE THREAD

**Issue by Stompmaster** [cmdrmcdonald/EliteDangerousDataProvider#279]
Hi I cant get any voice from EDDI ver 2.3.0

**Comment by Tkael** [comment]
is this a new issue / was it working before? have you already had a look at https://github.com/EDCD/EDDI/blob/master/TROUBLESHOOTING.md? have you already checked your system settings to ensure that sounds are not muted, either for this application or overall? we need more detail before we can help.
